### PR TITLE
fix(core): remove quotes inside `url(*)` function in fonts.less

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: projects/core/styles/taiga-ui-fonts.less [hunk 1]
```diff
@@ -1 +1 @@
-@import (css) url('https://fonts.googleapis.com/css2?family=Manrope:wght@500;800&display=swap');
+@import (css) url(https://fonts.googleapis.com/css2?family=Manrope:wght@500;800&display=swap);
```
Comment (nsbarsukov) [inline projects/core/styles/taiga-ui-fonts.less:1]: Lint fails with error:

```
 1:19  ✖  Expected quotes around "url" function argument  function-url-quotes
```

Comment (splincode) [reply projects/core/styles/taiga-ui-fonts.less:1]: Oh, no

Comment (splincode) [reply projects/core/styles/taiga-ui-fonts.less:1]: fixed

File: projects/core/styles/taiga-ui-fonts.less [hunk 2]
```diff
@@ -1 +1,3 @@
-@import (css) url('https://fonts.googleapis.com/css2?family=Manrope:wght@500;800&display=swap');
+// stylelint-disable function-url-quotes
```
Comment (nsbarsukov) [inline projects/core/styles/taiga-ui-fonts.less:1]: Maybe it is better to disable this stylint-rule ?

`url(...)` is used only in 2 `.less`-files of our project (50% does not satisfy this rule :peka:)

Comment (nsbarsukov) [reply projects/core/styles/taiga-ui-fonts.less:1]: nitpicking.

it is just up to you. You can keep it in this way.

**Review by nsbarsukov — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

File: projects/core/styles/taiga-ui-fonts.less (hunk 2 above)
Comment (splincode) [reply projects/core/styles/taiga-ui-fonts.less]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**